emily/handler/src/api/routes/mod.rs

```diff
@@ -21,29 +21,25 @@ mod withdrawal;
 #[cfg(feature = "testing")]
 pub fn routes(
     context: EmilyContext,
-) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
-    // TODO(273):  Remove the "local" prefix once we figure out why all local
-    // testing calls seem to forcibly start with `local`.
-    warp::path("local").and(
-        health::routes()
-            .or(chainstate::routes(context.clone()))
-            .or(deposit::routes(context.clone()))
-            .or(withdrawal::routes(context.clone()))
-            .or(testing::routes(context)),
-    )
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+    health::routes()
+        .or(chainstate::routes(context.clone()))
+        .or(deposit::routes(context.clone()))
+        .or(withdrawal::routes(context.clone()))
+        .or(testing::routes(context))
+        // Convert reply to tuple to that more routes can be added to the returned filter.
+        .map(|reply| (reply,))
```

Comment:
Not sure that I understand the tuple change. Curious, what is going on here?

Reply:
Warp filters all expect to take the output of the previous filter as the input to the next one, but it's implemented where each filter actually takes in the previous arguments as a tuple where the individual elements of the tuple are the arguments to the function.

This logic doesn't work if the output isn't a tuple because tuples can be turned into itemized function variables, but a single output cannot. Replying with a reply as a single element tuple allows it to be provided as a function argument :)